cloudpickle/cloudpickle.py

```diff
@@ -78,6 +78,7 @@
     PY3 = True
 
 
+
```

Comment:
Please do no insert an extra blank line. PEP8 only require 2 blank lines to separate top level blocks.